apis/v1alpha1/httproute_types.go

```diff
@@ -86,19 +86,35 @@ type HTTPRouteRule struct {
 	// incoming HTTP requests.
 	// Each match is independent, i.e. this rule will be matched
 	// if **any** one of the matches is satisfied.
+	//
 	// For example, take the following matches configuration:
+	//
+	// ```yaml
```

Comment:
This feels strange to see in go comments, does this end up helping the reference doc?

Reply:
Unfortunately not. I haven't been able to get the API doc generator to emit reformatted text of any kind. I can just remove this, since it doesn't really help.

Reply:
I removed the 'yaml', but kept the markdown formatting so that it formats as a block in the API docs (indentation is still wrong though).